### DISCUSSION TITLE
Kindle like reading experience

### DISCUSSION THREAD

**Discussion by RikardSwed** [RyotaUshio/obsidian-pdf-plus#166] [Q&A]
Great plugin. 

However, especially on mobile it would be nice to be able to read pdf's in a way that is similar to kindle. meaning you just choose a font size and then you start reading and the text automatically covers the whole screen. No need to zoom or swipe around to see and read things, the right amount of words cover the screen and when you have finished reading one page you swipe to the next. of course the features of highlighting text that is now in pdf++ should be intact in some kind of way,

**Comment by RyotaUshio** [comment]
That's not how PDF files work. I recommend googling something like "pdf vs epub", which will explain to you the difference between the PDF file format and ebook formats like EPUB.

**Reply by RyotaUshio** [reply]
If you want that kind of experience, one way to go would be to use [Calibre](https://calibre-ebook.com/) to convert PDF into EPUB and then use your favorite ebook reader.

**Reply by RikardSwed** [reply]
Sure but I really like your plugin and want to keep all the features. I understand that I can even put my pdf's into Kindle and use kindle to highlight, but I don't get those bidirectional links that I get with pdf++. It would just be great to be able to stay within obsidian to do this. 

Maybe it is not possible at all. But if it is it  would be nice. Also I just want it as a feature, a button to press that change the view to this. I don't always want to read in this way.  

Like I mentioned, a feature like this would make it possible to use the app on a mobile. which I would argue that it isn't really now, because you need to scroll around and change the zoom to be able to read. But even on a tablet, it would be nice to be able to change to this epub kind of reading style. 

If the file format is the issue. Would it be possible to update pdf++ so that it reads epub as well. with its current features intact?

**Reply by RyotaUshio** [reply]
Unfortunately, ebook-like reading styles will never be possible as long as we are handling PDF files. It is a restriction that comes from the PDF file format itself, not the design of PDF viewer apps.

> Would it be possible to update pdf++ so that it reads epub as well. with its current features intact?

Yeah that's what I'm also hoping to realize at some point, and there is a feature request for that: #63

**Reply by RikardSwed** [reply]
well, now we are talking then  😊 Keep up the good work!
